recipes/simple-salesforce/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: simple-salesforce
+  version: "0.72.2"
+
+source:
+  fn: simple-salesforce-0.72.2.tar.gz
+  url: https://pypi.python.org/packages/fa/f6/84ac2f3cc92df3ab5debb25730e4f3bf65be0d62c5140e22b46eafaaed07/simple-salesforce-0.72.2.tar.gz
```

Comment:
We prefer that the url should be replaced with the canonical form found in:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18-L19

and the `md5` should be replaced with the `sha256` (`b8a4cc544282be1da41e1ef474c1cbf4368ebbc24f25df93e4bd84bf9a1bfea9`).